File: csharp/eliuds-eggs/EliudsEggs.cs
public static class EliudsEggs
{
    public static int EggCount(int encodedCount)
    {
        string binaryString = Convert.ToString(encodedCount, 2);
        int count = 0;
        foreach (char bit in binaryString)
        {
            if (bit == '1')
            {
                count++;
            }
        }
        return count;
    }
}

File: javascript/eliuds-eggs/eliuds-eggs.js
//
// This is only a SKELETON file for the 'Eliud's Eggs' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const eggCount = (displayValue) => {
  // let count = 0;
  // while (displayValue > 0){
  //   count += (displayValue & 1) //Check if the bit is '1' or not
  //   displayValue >>= 1; //Shift to right to check next bit
  // }
  // return count
  return displayValue.toString(2).split('0').join('').length;
};
